styles/stylesheet.css

```diff
@@ -0,0 +1,304 @@
+* {
+  box-sizing: border-box;
+}
+
+/* header */
+a {
+  text-decoration: none;
+  font-family: 'Open Sans';
+}
+h1, h2, h3 {
+  color: #224971;
+  font-family: 'Gelasio';
+}
+header {
+  width: 100%;
+  background-image: url("../images/header-image.jpg");
+  background-size:cover;
+}
+.header-content-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  padding: 4rem 5rem 6rem 5rem;
+}
+.checkout-button-content-container > a {
+  /* border: dashed red; */
+  padding: .5rem 2rem;
+  background-color: #95be66;
+  color: white;
+  font-weight: bold;
+  font-size: .85rem;
+}
+.checkout-button-content-container,
+.cart-content-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  /* border: dashed red; */
+}
+.cart-content-container img {
+  padding-right: .5rem;
+  justify-content: flex-end;
+}
+.checkout-content-container {
+  /* margin: 5rem; */
+  grid-column-start: 3;
+  /* border: dashed purple; */
+}
+header + nav {
+  width: 100%;
+  background-color: #e9e9e9;
+}
+nav > ul {
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+}
+header + nav > ul {
+  /* padding: 1.5rem 10rem; */
+  padding: 1% 10%;
+}
+nav ul li {
+  margin: 0;
+  list-style: none;
+  padding-right: .7rem;
+}
+header + nav a {
+  color: #224971;
+  font-family: 'Gelasio';
+  font-weight: bold;
+}
+li:nth-child(-n+2)::after {
+  padding-left: .7rem;
+  content: "/";
+}
+.logo-content-container > img {
+  margin: 0;
+  display: block;
+  width: 100%;
+}
+.logo-content-container > p {
+  color: #95be66;
+  margin: 0;
+  display: block;
+  font-family: 'Gelasio';
+  font-style: italic;
+  font-weight: 500;
+  font-size: 1.5rem;
+}
+.cart-content-container > p {
+  color: white;
+}
+.cart-content-container > img {
+  width: 7%;
+  min-width: 25px;
+}
+.main-content-container {
+  /* margin: 1.5rem 10rem; */
+  margin: 2% 10%;
+}
+
+/* home + dog */
+main p {
+  font-family: "Open Sans";
+}
+.intro img {
+  float: left;
+  margin: 0 2rem 0 0;
+  width: 25%;
+}
+.intro p {
+  font-weight: 300;
+  line-height: 150%;
+}
+.see-more-dogs-container {
+  text-align: center;
+  margin: 3rem;
+}
+.see-more-dogs-container > a {
+  font-weight: 800;
+  padding: 1% 3%;
+  border-style: none;
+  border-radius: 2px;
+  color: #224971;
+  background-color: #e9e9e9;
+}
+.dog-cards {
+  padding-top: 2rem;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 3rem;
+  clear: both;
```

Comment:
this clear might not be needed, especially if you're not using floats in this section